### PR TITLE
Fix nav title issue

### PR DESCRIPTION
**Pull request by Priva28**
accidentally committed to wrong branch

### CODE REVIEW

File: Project SF/Views/Tabs/Settings/ProfileSettings.swift [hunk 1]
```diff
@@ -16,56 +16,59 @@ struct ProfileSettings: View {
...
-                // TODO: Present image search through the gallery.
-            }, label: {
-                Image(systemName: "person.crop.circle.badge.plus")
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-            })
-            .frame(height: 100)
-            .padding(.vertical)
+        VStack(spacing: 0) {
+            Text("")
```
Comment (wfltaylor) [inline Project SF/Views/Tabs/Settings/ProfileSettings.swift:22]: Maybe add a comment describing why this is here?

**Review by wfltaylor — APPROVED** [review]

**Comment by RomanEsin** [comment]
> accidentally committed to wrong branch

No problem

**Comment by wfltaylor** [comment]
We should probably add xcuserdata to the gitignore

**Comment by Priva28** [comment]
> We should probably add xcuserdata to the gitignore

yeah definitely

**Review by RomanEsin — APPROVED** [review]

**Review by RomanEsin — APPROVED** [review]

**Pull request merged**